sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ResourceWrapper.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
```

Comment:
classes in `com.azure.cosmos.models` are part of our public api.
Does `ResourceWrapper` have to be in `com.azure.cosmos.models` ? can we move it to implementation?

Reply:
Put it in models for the following reason:
Since we moved Resource class into implementation, so some access modifier has been changed from package private to public. So if I put ResourceWrapper into implementation, then the getResource method will change to public, then will expose methods to client which we did not before. 

But I can definitely remove getResource() from the interface.

Reply:
Put in the models with the following thinking:

Since we moved Resource into implementation, so some method has changed from private package to public.  If I remove ResourceWrapper to implementation, then the getResource() will change to public which may expose methods we did not/should not? 

But it can be solved by two ways: (if move to implementation)
1. remove getResource() method
2. change to protected, have some implmentionBridge class to use this method from other places

thoughts?

Reply:
Having ResourceWrapper in public should be fine in my opinion if it is not leaking any Jackson libraries. 
No need to do extra work through *BridgeInternals if not required. 
@moderakh  thoughts ?